src/Resources/skeleton/validator/Validator.tpl.php

```diff
@@ -11,6 +11,10 @@ public function validate($value, Constraint $constraint)
     {
         /* @var $constraint \<?= $constraint_class_name ?> */
 
+        if (null === $value || '' === $value) {
+            return;
+        }
+
```

Comment:
should we also add a `// TODO implement the validation here` so that they know where to put it ?

Reply:
@stof, I think you're right! I've added a TODO entry.